tests/clients/test_user.py

```diff
@@ -11,68 +11,118 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 import uuid
+from datetime import datetime
 
 import pytest
 from marshmallow import ValidationError
+from dateutil.tz import UTC
```

Comment:
`from dateutil.tz import UTC` should be before `from marshmallow import ValidationError`.